d2l-table-style.js

```diff
@@ -6,414 +6,606 @@ const $_documentContainer = document.createElement('template');
 $_documentContainer.innerHTML = `<dom-module id="d2l-table-style">
 	<template>
 		<style>
-			.d2l-table {
+			d2l-table-wrapper[type="default"] .d2l-table,
+			d2l-table-wrapper[type="light"] .d2l-table {
```

Comment:
Was the extra specificity required here? If the mixin needs to apply in both cases, could it not have stayed as `.d2l-table`?

Reply:
After adding the differentiator rules just to the places that needed it, I was seeing a lot of places where the styles that used to win were losing out and many rules were broken.  I figured it would be safest to add the extra specificity to every rule to bring them all back to the same level, but it does look like quite a mess now.  If you're talking specifically about this first set, it probably didn't need it.

Reply:
I can go back through and be more deliberate about which I think need it and which don't.  Do we see this as a major change?  In theory, it shouldn't break anything, but the specificity problems I was seeing were hover/select issues and issues with putting t* elements in a table cell (which I'm fairly sure rubrics does).

Reply:
Cool yeah I figured it was that. Like ideally rules that we want to apply to all styles shouldn't need anything, but with so MUCH CSS in here I get that in practice that's pretty hard.

It should be a minor release if we can -- hopefully there's time to catch any regressions?